browser/resources/settings/brave_wallet_page/brave_wallet_page.ts

```diff
@@ -180,6 +223,19 @@ class SettingsBraveWalletPage extends SettingsBraveWalletPageBase {
     this.browserProxy_.resetTransactionInfo()
     window.alert(this.i18n('walletResetTransactionInfoConfirmed'))
   }
+
+  onClearPinnedNftTapped_() {
+    if (this.pinnedNftCount_ == 0) {
+      return
+    }
+    var message = this.i18n('walletClearPinnedNftConfirmation')
```

Comment:
I don't see that we are putting in the translated confirmation phrase in this string. Is it just displaying $1 when you use this, instead of "Yes"?

Reply:
This is added in brave_settings_localized_strings_provider.cc

Reply:
In brave_settings_localized_strings_provider.cc I only see:
```
    {"walletClearPinnedNftConfirmation",
     IDS_SETTINGS_WALLET_CLEAR_PINNED_NFT_CONFIRMATION},
```

And no handling of the placeholder
```
Pinned NFTs are hosted to the IPFS network when local IPFS node is launched.
    You have <ph name="NFT_COUNT">$1<ex>2</ex></ph> pinned NFTs at the moment.
```

Am I missing something?

Reply:
```
  {"walletClearPinnedNftConfirmation",
     IDS_SETTINGS_WALLET_CLEAR_PINNED_NFT_CONFIRMATION},
```
Actually should be deleted cause there is  
```

 html_source->AddString("walletClearPinnedNftConfirmation",
                         clear_pinned_nft_confirmation);
```